Docker/Invoke-Build.ps1

```diff
@@ -223,6 +223,12 @@ function Invoke-Build {
         $stdVer = ""
     }
 
+    #building the images from a branch different than main, will add the package version suffix to the pre tag
+    $preTag = "pre"
+    if ($env:BASE_BRANCH -ne 'main') {
+      $preTag = $preTag+$PackageVersion.Replace(".", "")
```

Comment:
Add a `-` after the `pre` word. Right now it would generate a tag name like `pre71` which doesn't follow our existing convention.